intro-HPC/ch_preparing_the_environment.tex

```diff
@@ -464,6 +757,21 @@ \subsection{Activating and de-activating modules}
 
 This will load the two modules as well as their dependencies.
 
+\ifgent
+Finally, to switch between clusters you can swap the ``cluster'' modules.
+These are special modules that change what modules are available for you,
+and what cluster your jobs will be queued in.
+By default you are working on \hpcname. To switch to e.g. raichu you need:
```

Comment:
rephrase to make it very clear that your only redefining your environment to i) prepare for submitting jobs to the cluster you specified, ii) get access to modules that are available on that cluster (but shouldn't be used directly on the login nodes). After swapping the `cluster` module, you'll still be logged in on the login node.

This is a common misconception among new users, so it's important to explain this part well.